libs/ragulate/ragstack_ragulate/cli.py

```diff
@@ -1,5 +1,4 @@
 # this must be imported first for dynamic module loading to function properly
-import setuptools  # isort: skip
```

Comment:
we need to keep this one

Reply:
@nicoloboschi there is prob a way for ruff to ignore lines

Reply:
added back